### ISSUE TITLE
kubernetes provider: No Auth Provider found (GCP)

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-kubernetes#5]
_This issue was originally opened by @mattes as hashicorp/terraform#15244. It was migrated here as part of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

### Terraform Version
Terraform v0.9.8

### Affected Resource(s)
- kubernetes

### Terraform Configuration Files
```hcl
provider "kubernetes" {}

resource "kubernetes_namespace" "example" {
  metadata {
    name = "my-first-namespace"
  }
}
```

### Debug Output
```
[truncated]
2017/06/10 02:12:01 [DEBUG] plugin: terraform: kubernetes-provider (internal) 2017/06/10 02:12:01 [INFO] Successfully loaded config file (/Users/matthias/.kube/config; default context)
2017/06/10 02:12:01 [ERROR] root: eval: *terraform.EvalConfigProvider, err: Failed to configure: No Auth Provider found for name "gcp"
2017/06/10 02:12:01 [ERROR] root: eval: *terraform.EvalSequence, err: Failed to configure: No Auth Provider found for name "gcp"
2017/06/10 02:12:01 [ERROR] root: eval: *terraform.EvalOpFilter, err: Failed to configure: No Auth Provider found for name "gcp"
2017/06/10 02:12:01 [ERROR] root: eval: *terraform.EvalSequence, err: Failed to configure: No Auth Provider found for name "gcp"
2017/06/10 02:12:01 [TRACE] [walkPlan] Exiting eval tree: provider.kubernetes
2017/06/10 02:12:01 [DEBUG] dag/walk: upstream errored, not walking "kubernetes_namespace.example"
2017/06/10 02:12:01 [DEBUG] dag/walk: upstream errored, not walking "meta.count-boundary (count boundary fixup)"
2017/06/10 02:12:01 [DEBUG] dag/walk: upstream errored, not walking "provider.kubernetes (close)"
2017/06/10 02:12:01 [DEBUG] dag/walk: upstream errored, not walking "root"
[31mError running plan: 1 error(s) occurred:

2017/06/10 02:12:01 [DEBUG] plugin: waiting for all plugin processes to complete...
* provider.kubernetes: Failed to configure: No Auth Provider found for name "gcp"[0m[0m
2017/06/10 02:12:01 [DEBUG] plugin: terraform: kubernetes-provider (internal) 2017/06/10 02:12:01 [DEBUG] plugin: waiting for all plugin processes to complete...
2017/06/10 02:12:01 [DEBUG] plugin: /usr/local/Cellar/terraform/0.9.8/bin/terraform: plugin process exited
```

### Error Output
```
provider.kubernetes: Failed to configure: No Auth Provider found for name "gcp"
```

I tried several options, even with statically defined credentials the same error is thrown.

**Comment by svenmueller** [comment]
i'm running version `Terraform v0.9.11` and still get the following error. Any idea how to solve this?

`* module.kubernetes.provider.kubernetes: Failed to configure: No Auth Provider found for name "gcp"`

**Comment by radeksimko** [comment]
@svenmueller per [changelog](https://github.com/terraform-providers/terraform-provider-kubernetes/blob/master/CHANGELOG.md) this bugfix was released in `0.1.1` which is available via any of the `0.10` recent beta Terraform releases.

There are no plans of backporting this to `0.9`.
